Get-NetView.psm1

```diff
@@ -2605,7 +2614,11 @@ function Get-NetView {
         [Int] $BackgroundThreads = 5,
 
         [parameter(Mandatory=$false)]
-        [Switch] $SkipAdminCheck = $false
+        [Switch] $SkipAdminCheck = $false,
+
+        [parameter(Mandatory=$false)]
+        [ValidateSet("Universal", "AzureHost", "FailoverCluster")]
+        [String] $Scenario = "Universal"
```

Comment:
See comments above on scenario removal.

Reply:
I'll probably replace "AzureHost" with a `-NonDisruptive` switch parameter that is mutually exclusive with `-BackgroundThreads`

Reply:
Sure.  But consider the operationalization options here.
Using all cores where each has the -NonDirsuptive behavior could be useful to get both non-disruption and faster completion through parallelism.